src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,44 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
-    private final StorageDao storageDao = new StorageDaoImpl();
+    private static final int MIN_AGE = 18;
+    private static final int MIN_PASSWORD_AND_LOGIN_LENGTH = 6;
```

Comment:
Let's separate them: `MIN_PASSWORD_SIZE` and `MN_LOGIN_SIZE`. The requirements for min length can change in the future, so we need to separate them.